test/test_socketcan.py

```diff
@@ -353,6 +355,21 @@ def test_build_bcm_update_header(self):
         self.assertEqual(can_id, result.can_id)
         self.assertEqual(1, result.nframes)
 
+    @unittest.skipUnless(IS_LINUX and IS_PYPY, "Only test when run on Linux with PyPy")
+    def test_pypy_socketcan_support(self):
+        """Wait for PyPy raw CAN socket support
+
+        This test shall document raw CAN socket support under PyPy. Once this test fails, it is likely that PyPy
+        either implemented raw CAN socket support or at least changed the error that is thrown.
+        https://foss.heptapod.net/pypy/pypy/-/issues/3809
+        https://github.com/hardbyte/python-can/issues/1479
+        """
+        try:
+            can.Bus(interface="socketcan", channel="vcan0", bitrate=500000)
+        except OSError as e:
+            if "unknown address family" not in str(e):
+                raise
```

Comment:
This test is a good idea, but let's change this to a warning message instead of a failing CI.